comex-main/src/main/java/br/com/alura/comex/Main.java

```diff
@@ -0,0 +1,38 @@
+package br.com.alura.comex;
+
+import java.math.RoundingMode;
+import java.text.NumberFormat;
+import java.util.*;
+
+import br.com.alura.comex.processadores.ProcessadorDeCsv;
+import br.com.alura.comex.relatorios.RelatorioSintetico;
+
+public class Main {
+
+	public static void main(String[] args) {
+
+		ProcessadorDeCsv processadorDeCsv = new ProcessadorDeCsv();
+		String arquivoCSV = "pedidos.csv";
+		List<Pedido> listaPedidos;
+		listaPedidos = processadorDeCsv.listarPedidos(arquivoCSV);
```

Comment:
Pode fazer juntar a declaração dessa variável!

```suggestion
		List<Pedido> listaPedidos = processadorDeCsv.listarPedidos(arquivoCSV);
```